hail/src/main/scala/is/hail/backend/service/Worker.scala

```diff
@@ -158,11 +158,11 @@ object Worker {
     timer.start("executeFunction")
 
     if (HailContext.isInitialized) {
-      HailContext.get.backend = new ServiceBackend(null, null, new HailClassLoader(getClass().getClassLoader()), null, None)
+      HailContext.get.backend = new ServiceBackend(null, null, new HailClassLoader(getClass().getClassLoader()), null, None, null, null, null, null)
```

Comment:
This is feeling increasingly wrong. I think after this PR lands I'll PR something that creates a `QoBWorkerBackend` which is just the class loader and raises UnsupportedOperationException for everything else.

Reply:
that would be great